src/const/regions.ts

```diff
@@ -0,0 +1,3647 @@
+import { Region } from '../interfaces/wine.interface';
+
+export const REGIONS: Region[] = [
```

Comment:
Regions API is taking a lot of time to get the response in vivino website. That's why I kept it as a file so that our Node project will be fast as we are not waiting for that API to be fetched. 
Also we don't need Regions API to get the wines list unless the user is providing a region with the command.

I think a better approach would be scrapping Regions API if the user has inputted the region and not use the hardcoded regions.